src/cli/utils/logger.spec.ts

```diff
@@ -32,4 +32,39 @@ describe('cli/utils/logger', () => {
       },
     );
   });
+
+  test('works - stdout with json', async () => {
+    env.CONTAINERBASE_LOG_FORMAT = 'json';
```

Comment:
please delete these new variables like the others in `beforeEach`

Reply:
done